File: 595-Big-Counties-EASY/soln.py
import pandas as pd

def big_countries(world: pd.DataFrame) -> pd.DataFrame:
    big_area_mask = world['area'] >= 3000000
    big_population_mask = world['population'] >= 25000000

    cols = ['name', 'population', 'area']
    return world.loc[big_area_mask | big_population_mask, cols]

File: 595-Big-Counties-EASY/soln.scala
import org.apache.spark.sql.DataFrame
import org.apache.spark.sql.functions.col

object Solution {
  def bigCountries(world: DataFrame): DataFrame = {
    world
      .filter(col("area") >= 3000000 || col("population") >= 25000000)
      .select("name", "population", "area")
  }
}
